oasislmf/utils/data.py

```diff
@@ -109,6 +111,29 @@
 # Load schema json dir
 SCHEMA_DATA_FP = os.path.join(os.path.abspath(os.path.join(os.path.dirname(__file__), os.pardir)), 'schema')
 
+RI_INFO_DEFAULTS = {
```

Comment:
Should the defaults not come from the OED spec, rather than be hard-coded in oasislmf?